kubeflow-pipelines/docling-vlm/README.md

```diff
@@ -0,0 +1,24 @@
+# Docling and Kubeflow Pipelines Demo
+
+## Running the demo pipeline directly
+
+Assuming you have a working Kubeflow Pipelines installation, download [docling_pipeline.yaml](docling_pipeline.yaml) and the upload it to your Kubeflow Pipelines UI or via the CLI. Then, start a run of that pipeline to see the conversion process happen.
```

Comment:
Minor fix, the name of the YAML we're compiling is now `docling_convert_pipeline_compiled.yaml`.